README.md

```diff
@@ -155,55 +169,64 @@ Example:
     }
   }}
 >
-...
+{/* ... */}
```

Comment:
Same thing here.  The intention here is just to show child elements would still be necessary here but highlighting the portion being written about. So this already can't be copy-pasted anywhere so I'm not too worried about it being valid jsx and was fine how it was written before

Reply:
I am OK in undo those changes if you think this is not necessary.